### PR TITLE
Cleaning up nosekatreport before migrating to Python 3.

### PR DESCRIPTION
**Pull request by mmphego**
This PR does the following:

- [x] Replaces `MIT`/`GPL` license with `BSD` as per requirement.
- [x] Added katversion as version number maintainer.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
-# SKA South Africa (http://ska.ac.za/)                                        #
-# Author: cam@ska.ac.za                                                       #
-# Copyright @ 2013 SKA SA. All rights reserved.                               #
-#                                                                             #
-# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
-# WRITTEN PERMISSION OF SKA SA.                                               #
-###############################################################################
-import sys
-
+# Copyright 2013 SKA South Africa (http://ska.ac.za/)
```
Comment (tockards) [inline setup.py]: Not entirely sure what the license is  as I am also adding a license in the https://github.com/ska-sa/katportalclient/pull/50 PR.

Comment (mmphego) [reply setup.py]: :man_facepalming: talk about copy and paste, at least the `LICENSE` file is pointed out in the comment.

Comment (tockards) [reply setup.py]: Sorry, I need to perfect my written word. 
I am not entirely sure if the line 
`Copyright 2013 SKA South Africa (http://ska.ac.za/)` is correct. 
As we are no longer `SKA South Africa`  but @ajoubertza answered it over here. 
https://github.com/ska-sa/katportalclient/pull/50/files#r319179863

File: setup.py [hunk 2]
```diff
@@ -1,24 +1,15 @@
...
+    description='Nose plugin for writing an annotated test report',
     long_description=open('README.rst').read(),
-    author='Neilen Marais',
-    author_email='nmarais@ska.ac.za',
-    license='GPL',
+    author='cam',
+    author_email='cam@ska.ac.za',
+    license='BSD',
     packages=find_packages(exclude=['ez_setup']),
     install_requires=['Nose>=0.11.0', 'traceback2', 'ansicolors', 'numpy'],
```
Comment (amakhaba) [inline setup.py:14]: Shouldnt we also specify which version of `numpy` to install ?

Comment (mmphego) [reply setup.py:14]: No need to pin it. It is only being used in https://github.com/ska-sa/nosekatreport/blob/9a202709f7c7a1dc4af52ed0981a8830355ac24a/nosekatreport/plugin.py#L936+L960 which is mostly used by **CBF**. 

**Note:** Python2 support ceases from version `1.16.5`

File: LICENSE [hunk 3]
```diff
@@ -1,21 +1,27 @@
-MIT License
+Copyright (c) 2013 SKA South Africa
```
Comment (ajoubertza) [inline LICENSE]: `Copyright (c) 2017 National Research Foundation (South African Radio Astronomy Observatory)`

File: nosekatreport/__init__.py [hunk 4]
```diff
@@ -1,2 +1,19 @@
+# Copyright (c) 2013 National Research Foundation (South African Radio Astronomy Observatory)
```
Comment (ajoubertza) [inline nosekatreport/__init__.py]: The year must match when the file was added.  These files were only added in 2017.

File: nosekatreport/decorators.py [hunk 5]
```diff
@@ -1,17 +1,9 @@
-###############################################################################
-# SKA South Africa (http://ska.ac.za/)                                        #
-# Author: cam@ska.ac.za                                                       #
-# Copyright @ 2013 SKA SA. All rights reserved.                               #
-#                                                                             #
-# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
-# WRITTEN PERMISSION OF SKA SA.                                               #
-###############################################################################
+# Copyright (c) 2013 National Research Foundation (South African Radio Astronomy Observatory)
```
Comment (ajoubertza) [inline nosekatreport/decorators.py]: 2017

File: nosekatreport/decorators.py [hunk 6]
```diff
@@ -166,137 +149,6 @@ class TestGeneric(unittest.TestCase):
     """
     return grand_decorator('generic_test', *args)
 
-def instrument_bc8n856M4k(*args):
```
Comment (ajoubertza) [inline nosekatreport/decorators.py:169]: Has it been confirmed that these instrument decorators are not used by CBF?

File: nosekatreport/plugin.py [hunk 7]
```diff
@@ -1,6 +1,7 @@
+# Copyright (c) 2013 National Research Foundation (South African Radio Astronomy Observatory)
```
Comment (ajoubertza) [inline nosekatreport/plugin.py]: 2017

File: nosekatreport/tests/test_aqf.py [hunk 8]
```diff
@@ -1,13 +1,5 @@
...
-# SKA South Africa (http://ska.ac.za/)                                        #
-# Author: cam@ska.ac.za                                                       #
-# Copyright @ 2013 SKA SA. All rights reserved.                               #
-#                                                                             #
-# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
-# WRITTEN PERMISSION OF SKA SA.                                               #
-###############################################################################
-""" ."""
-
+# Copyright (c) 2013 National Research Foundation (South African Radio Astronomy Observatory)
```
Comment (ajoubertza) [inline nosekatreport/tests/test_aqf.py]: 2017

File: setup.py [hunk 9]
```diff
@@ -1,24 +1,15 @@
...
-# SKA South Africa (http://ska.ac.za/)                                        #
-# Author: cam@ska.ac.za                                                       #
-# Copyright @ 2013 SKA SA. All rights reserved.                               #
-#                                                                             #
-# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
-# WRITTEN PERMISSION OF SKA SA.                                               #
-###############################################################################
-import sys
-
+# Copyright (c) 2013 National Research Foundation (South African Radio Astronomy Observatory)
```
Comment (ajoubertza) [inline setup.py]: 2017

**Review by ajoubertza — CHANGES_REQUESTED** [review]

File: nosekatreport/decorators.py (hunk 6 above)
Comment (mmphego) [reply nosekatreport/decorators.py:169]: CBF uses this branch: https://github.com/ska-sa/nosekatreport/tree/karoocbf
Any further development is being done in the karoocbf branch.

**Review by ajoubertza — APPROVED** [review]
LGTM

**Review by tockards — APPROVED** [review]
LGTM

**Pull request merged**